app/controllers/api/v1/rents_controller.rb

```diff
@@ -1,15 +1,20 @@
 module Api
   module V1
-    class RentsController < ApplicationController
+    class RentsController < ApiController
+      rescue_from Pundit::NotAuthorizedError, with: :pundit_not_authorized_error
       include Wor::Paginate
 
       def index
         @rents = Rent.includes(:book, :user).where(user_id: params[:id])
+        @user  = User.find(params[:user_id])
+        authorize(@user)
         render_paginated @rents
       end
 
       def create
         @rent = Rent.new(rent_params)
+        @user = User.find(params[:user_id])
+        authorize(@user)
```

Comment:
When you are creating/updating or removing a rent, you should authorize the element you are authorizing.

This should be `authorize(@rent)`.
Also you should use the devise utilities to get the current user (you can just write `@user = current_user`)

Reply:
Done